firstvoices/pytest.ini

```diff
@@ -1,3 +1,5 @@
 [pytest]
 addopts = --ds=firstvoices.settings_test --create-db -nauto
 python_files = tests.py test_*.py
+markers =
+    disable_thumbnail_mocks: disable generation of mock thumbnails
```

Comment:
This flag actually does the opposite. When the thumbnail mocks are enabled, nothing is generated. When the mocks are disabled, the real thumbnails are generated. So this comment should be more like "enable thumbnail generation, by disabling the mocks that prevent it"